### DISCUSSION TITLE
Undefined Error on Register

### DISCUSSION THREAD

**Discussion by itcontrol-ca** [socioboard/Socioboard-5.0#428] [Q&A]
I've installed Socioboard 5.0 on an AWS EC2 Instance (t3.small). Ubuntu 22.04 
I'm able to access the site, but unable to register an account.
All services are running (Users, Feeds, etc).

When I go to registered, I get an undefined error. in the configs I see host to localhost for all microservices. 

What am I missing?

**Comment by sureshbabu1995** [comment]
Hi @itcontrol-ca 

If you're using any domains, it's better to update the localhost pointing to the domains you are using, for the sample purpose and local run we kept the domains as localhost.
